### ISSUE TITLE
Crafty+

### ISSUE THREAD

**Issue by iMonZ** [ImACoderImACoderImACoder/onyx#28]
It would be awesome seeing crafty+ support.
The JavaScript code from S&B Webapp is easily reverse engineerable

**Comment by ImACoderImACoderImACoder** [comment]
I agree crafty+ support would be awesome, however I don't have a crafty and that makes it too difficult for me to ensure the quality of the application.
